src/wmtk/operations/tet_mesh/TetSplitWithTags.hpp

```diff
@@ -19,8 +19,10 @@ struct OperationSettings<tet_mesh::TetSplitWithTags>
     MeshAttributeHandle<double> pos_handle;
     long split_vertex_tag_value;
 
+    OperationSettings(const Mesh& m): invariants(m) {}
```

Comment:
the operationsettings objects now need to default to default initialize the invariants with a mesh - lets just put this in headers for now to be lazy

Reply:
(dont do this if you want to use shared_ptr)